upstream-manifests/federation/0.0.10/federation.v0.0.10.clusterserviceversion.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     description: Gain Hybrid Cloud capabilities between your clusters with Kubernetes Federation.
     certified: "false"
     containerImage: "quay.io/kubernetes-multicluster/federation-v2:v0.0.10"
-    createdAt: "2019-01-01T00:00:00Z"
+    createdAt: "2019-05-17T09:00:00Z"
```

Comment:
The date of this image should actually be May 6 2019 6:56 pm according to the tag history on quay: https://quay.io/repository/kubernetes-multicluster/federation-v2?tab=tags

Reply:
Fixed